app/views/appeal/tree.scala

```diff
@@ -311,10 +288,10 @@ object tree:
       )
     )
 
-  private def playbanMenu: Branch =
+  private def playbanMenu(using PageContext): Branch =
```

Comment:
`using Context` should be enough here, as this function doesn't render a full page.